rotkehlchen/exchanges/manager.py

```diff
@@ -306,11 +308,21 @@ def get_user_binance_pairs(self, name: str, location: Location) -> list[str]:
             return self.database.get_binance_pairs(name, location)
         return []
 
-    def query_history_events(self) -> None:
-        """Queries all history events for exchanges that need it
+    def query_history_events(self, name: str, location: Location) -> None:
+        """Queries new history events for the specified exchange.
 
         May raise:
-        - RemoteError if any exchange's remote query fails
+        - RemoteError if the exchange's remote query fails
         """
-        for exchange in self.iterate_exchanges():
-            exchange.query_history_events()
+        if (exchange := self.get_exchange(name=name, location=location)) is None:
+            log.error(
+                'Failed to query history events for unknown exchange. '
+                f'Location: {location!s}, Name: {name}',
+            )
+            return
+
+        exchange.query_history_events(
+            start_ts=Timestamp(0),
```

Comment:
Hmmm shouldn't it query from latest event until now? Or is this logic implemented inside the function? If so then why do you need the start_ts passed as an argument here? I am a bit confused. Doesn't make much sense to me.

Would you not want to query just query_online_history_events here?

Reply:
Simplified the `query_history_events` function, so now no arguments are passed here. The other functions similar to `query_history_events` in exchange interface should be simplified as well, but leaving that for now as discussed in discord.

Note that a couple tests had a check for the ability to query specific ranges via query_history_events (or a function it replaces). So these checks have been removed.

> Would you not want to query just query_online_history_events here?

It does need to use the query ranges logic in query_history_events so no, directly calling query_online_history_events here wouldn't work.